resources/local/chainspec.toml.in

```diff
@@ -11,7 +11,7 @@ hard_reset = false
 # in contract-runtime for computing genesis post-state hash.
 #
 # If it is an integer, it represents an era ID, meaning the protocol version becomes active at the start of this era.
-activation_point = '${TIMESTAMP}'
+activation_point = '2024-03-04T20:59:06.516234Z'
```

Comment:
This looks like a mistake - it should remain just a template, shouldn't it?

Reply:
Fixed